settings.gradle

```diff
@@ -47,4 +47,5 @@ include 'e2e-test'
 include 'e2e-test:trace'
 include 'e2e-test:log'
 include 'data-prepper-test-common'
+include 'data-prepper-performance-test'
```

Comment:
Could we just make this module name `performance-test`? That would follow the pattern of `e2e-test`. For the most part, `data-prepper-` projects produce library artifacts for general distribution.